### ISSUE TITLE
$this->validateOnly from Mount: Call to a member function toArray() on null: $this->errorBag->toArray()

### ISSUE THREAD

**Issue by jjjrmy** [livewire/livewire#1178]
I am getting an error from the ErrorBag when calling another function with validateOnly.

```php
Call to a member function toArray() on null
vendor/livewire/livewire/src/ComponentConcerns/ValidatesInput.php
---
$this->setErrorBag(
    Arr::except($this->errorBag->toArray(), $field)
 );
```

Here is a simplified version of my component:
```php
public $a;

public function mount() {
    if(true) {
        $this->a = "##########";
        $this->validateOnly('a', ['a' => 'required'], ['a.required' => 'Fill this field please.']);
        // ^^^ This is what is causing the error!
        // Or if I were to do this instead:
        // $this->phone();
        // ^^^ This has the same error!
    }
}

public function phone() {
    $this->validateOnly('a', ['a' => 'required'], ['a.required' => 'Fill this field please.']);
    // ^^^ This errors only if triggered by mount()
    // blah blah blah
    // do more stuff
}
```

If you comment out the `$this->validateOnly` in mount it works fine, so that's definitely the issue.
If the conditional in mount is false and you trigger the `phone()` function from the View with `wire:click` it validates the input fine and runs the rest of the function fine.

The problem is only caused when running $this->validateOnly from mount.

**Context**
- Livewire version: [e.g. 1.2.0]
- Laravel version: [e.g. 7.0.0]

**Comment by jjjrmy** [comment]
Maybe this might fall under a new issue ...

If you add an error in the mount, it does not trigger.
```php
public function mount() {
    $this->addError('a', 'Some Message');
}
```

Also just as a test I did something like this and the error message is not updating even though the field value is updating.
```php
public function updatedA($value) {
    $this->addError('a', rand());
}
```

**Comment by jjjrmy** [comment]
Fixed by #1186
